buildcmake

```diff
@@ -261,6 +265,13 @@ done
 
 ############ End option parsing
 
+if [[ "$target" = "${target%AMPI-only}AMPI-only" ]]; then
```

Comment:
why wouldnt this just be `if [[ "$target" = "AMPI-only" ]]` ?

Reply:
For all-test-AMPI-only